eng/Versions.props

```
@@ -46,6 +46,7 @@
     <SystemCommandLineVersion>2.0.0-beta1.20468.1</SystemCommandLineVersion>
     <SystemCommandLineRenderingVersion>2.0.0-beta1.20074.1</SystemCommandLineRenderingVersion>
     <SystemMemoryVersion>4.5.4</SystemMemoryVersion>
+    <SystemTextEncodingsWebVersion>4.7.1</SystemTextEncodingsWebVersion>
```

You might need to bump to 4.7.2

---

See https://github.com/dotnet/announcements/issues/178

---

Thanks, looks like I can't read. 
```suggestion
    <SystemTextEncodingsWebVersion>4.7.2</SystemTextEncodingsWebVersion>
```